client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -71,9 +123,14 @@ export class CommentDetailAuthor extends Component {
 			authorIsBlocked,
 			authorUrl,
 			authorUsername,
-			blockUser,
+			canUserBlacklist,
+			currentUserEmail,
+			site,
 			translate,
 		} = this.props;
+
+		const showBlockUser = canUserBlacklist && !! authorEmail && ( authorEmail === currentUserEmail );
```

Comment:
I think this supposed to be `( authorEmail !== currentUserEmail )`?

Reply:
Ah shoot, fumbled when moving things around, thanks for catching it!